sample/F8/sponsors

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
mahika i know there arent any sponsers but still can you add fb as a sponser.
i was made to add mozilla for my mozilla event.